/**
 예외 처리
 - do, try, catch 구문을 사용하여 오류를 처리하고 관리할 수 있음
   - try 키워드를 사용하여 에러가 발생할 수 있는 코드를 실행하고, 에러가 발생하면 catch 블록에서 해당 에러를 처리함
   - catch 블록에서는 특정 오류만 처리할 수 있음
 - try?, try!와 같이 에러를 간편하게 처리할 수 있는 방법을 제공함
   - do, catch를 사용하지 않아도 됨
 - throws
   - 함수가 에러를 던질 수 있다는 것을 명시하기 위해 사용하며, 함수의 파라미터 뒤에 throws 키워드를 작성함
   - 함수 내부에서 에러가 발생하면 throw 키워드를 사용하여 에러를 던질 수 있음
   - 에러는 함수를 호출한 곳에서 처리해야 함
 */
